learning/tour-of-beam/frontend/lib/pages/tour/widgets/unit_content.dart

```diff
@@ -30,6 +32,76 @@ class UnitContentWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MarkdownBody(data: unitContent.description);
+    final textTheme = Theme.of(context).textTheme;
+    final markdownStyle = MarkdownStyleSheet(
+      h1: textTheme.headlineLarge,
+      h3: textTheme.headlineMedium,
+      p: textTheme.bodyMedium,
+      code: Theme.of(context)
+          .extension<BeamThemeExtension>()!
+          .codeRootStyle
+          .copyWith(
+            backgroundColor: BeamColors.transparent,
+            fontSize: BeamSizes.size12,
+          ),
+      codeblockDecoration: BoxDecoration(
+        // TODO(nausharipov): where to store these colors?
```

Comment:
`Beam*ThemeColors.codeBackground` are fine.

Reply:
We no longer have `BeamThemeColors` class. Do I need to recreate it?

Reply:
In the the comment below you mention `BeamThemeExtension.codeBackgroundColor`, so I'll go with it.

Reply:
Yes. I just mean that `BeamLightThemeColors.codeBackground` and `BeamDarkThemeColors.codeBackground` are fine as the color constants since you asked that in the comment.